python_modules/dagster/dagster/_core/workspace/context.py

```diff
@@ -610,10 +610,11 @@ def _load_location(self, origin: RepositoryLocationOrigin) -> WorkspaceLocationE
             repository_location=location,
             load_error=error,
             load_status=WorkspaceLocationLoadStatus.LOADED,
-            display_metadata=location.get_display_metadata()
+            display_metadata=(location.get_display_metadata()
             if location
-            else origin.get_display_metadata(),
+            else origin.get_display_metadata()),
             update_timestamp=time.time(),
+            image=location.container_image if location else None,
```

Comment:
This information is already in display metadata right? Whats the motivation for repeating it? Just that we want to respect it as not-just-for-display?

Reply:
It's not in display metadata in serverless, and I am plotting to use it for something other than just display